dts/bindings/interrupt-controller/cypress,psoc6-int-mux.yaml

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2018 Foundries.io
+# Copyright (c) 2020 ATL Electronics
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Cypress Interrupt Multiplex
+
+  The PSoC-6 Cortex-M0+ NVIC can handle up to 32 interrupts. This means that
+  user can select up to 32 interrupts sources from the 240 possible vectors
+  to be processed in the Cortex-M0+ CPU.
+
+  At CPU Sub System (CPUSS) there are 8 special registers to configure the
```

Comment:
Can you add a sentence or something here that the 'vector number' is that get placed into the mux field is the IRQ number for the NVIC to the Cortex-M4.

Reply:
Done!